### PR TITLE
[ticket/14201] Add multiple ACP template events

### PR DESCRIPTION
**Pull request by rxu**
<a href="https://tracker.phpbb.com/browse/PHPBB3-14201">PHPBB3-14201</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set WIP

**Comment by rxu** [comment]
!set Event

**Comment by rxu** [comment]
!unset WIP

File: phpBB/adm/style/acp_ban.html [hunk 1]
```diff
@@ -52,7 +52,7 @@
 	<legend>{L_TITLE}</legend>
 <dl>
 	<dt><label for="ban">{L_BAN_CELL}{L_COLON}</label></dt>
-	<dd><textarea name="ban" cols="40" rows="3" id="ban"></textarea></dd>
+	<dd><!-- EVENT acp_ban_cell_textarea_prepend --><textarea name="ban" cols="40" rows="3" id="ban"></textarea><!-- EVENT acp_ban_cell_textarea_append --></dd>
```
Comment (Nicofuma) [inline phpBB/adm/style/acp_ban.html]: I'd prefer an event with a more semantic name (not so tied to the html structure)

File: phpBB/adm/style/acp_email.html [hunk 2]
```diff
@@ -19,12 +19,14 @@
 	<legend>{L_COMPOSE}</legend>
 <dl>
 	<dt><label for="group">{L_SEND_TO_GROUP}{L_COLON}</label></dt>
+	<!-- EVENT acp_email_group_options_prepend -->
```
Comment (Nicofuma) [inline phpBB/adm/style/acp_email.html]: why do you need an event here? Could you give a use case?

Comment (rxu) [inline phpBB/adm/style/acp_email.html]: This is for adding additional html to implement a kind of the live search functionality (the ext is not mine though).

**Comment by rxu** [comment]
@Nicofuma Updated the event name for #3930.

**Comment by Nicofuma** [comment]
:+1:

**Pull request merged**